addons/certificate/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+certificate.access_certificate_admin,certificate.access_certificate,model_certificate_certificate,base.group_system,1,1,1,1
+certificate.access_certificate_user,certificate.access_certificate,model_certificate_certificate,base.group_user,1,0,0,0
+certificate.access_key_admin,certificate.access_key,model_certificate_key,base.group_system,1,1,1,1
+certificate.access_key_user,certificate.access_key,model_certificate_key,base.group_user,1,0,0,0
```

Comment:
Thanks a lot for your feedback! I'll do the changes as you mentioned. I was actually thinking about making private keys only accessible to admins and not user. I assumed that, since certificates are public by nature, it would not be an issue if users accessed them. But you're right, it's better to keep those as private as possible.